changelogs/fragments/1751-Avoid_parsing_on_job_utils.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - module_utils/job.py - Avoid parsing of job output and use new job type on ZOAU pytho API.
```

I don't think it is job type but rather job class. Also a typo 

```suggestion
  - module_utils/job.py - Avoid parsing of job output and use job class from ZOAU python API.
```